tests/test_filter_gammatone.py

```diff
@@ -206,3 +206,43 @@ def test_erb_frequencies_assertions():
     # resolution must be > 0
     with pytest.raises(ValueError, match="Resolution must be larger"):
         pf.dsp.filter.erb_frequencies([0, 1], 0)
+
+
+def test_impulse_response():
+    """Test the impulse response of GammatoneBands."""
+    GFB = pf.dsp.filter.GammatoneBands([0, 22050])
+    irs = GFB.impulse_response()
+    ir_real, ir_imag = irs
+
+    assert isinstance(irs, tuple)
+    assert len(irs) == 2
+    assert isinstance(ir_real, pf.Signal)
+    assert isinstance(ir_imag, pf.Signal)
```

Comment:
This is stricter and maybe more future proof
```suggestion
    assert type(ir_real) == pf.Signal
    assert type(ir_imag) == pf.Signal
```